### ISSUE TITLE
Scrobble event being created when pressing "Mark as Unread" on a completely new/unread series

### ISSUE THREAD

**Issue by chilliwow** [Kareadita/Kavita#2353]
**Describe the bug**
Scrobble event being created when pressing "Mark as Unread" on a completely new/unread series

**To Reproduce**
1. Go to a new series, (manga in my case)
2. Click on "Mark as Unread"
3. Go to scrobbling history
4. (Check screenshot)

**Expected behavior**
Not creating a scrobble event when just pressing "Mark as Unread" on an unread series.

**Screenshots**
![image](https://github.com/Kareadita/Kavita/assets/31785430/940336ef-e8fe-4e62-8ec1-b88c36478e30)

**Desktop:**
 - OS: Windows 11
 - Browser: Brave
 - Version: 0.7.9.0
 
**Additional context**
I'd like to add the reason why I pressed "Mark as Unread". Maybe you just checked out a few pages of a chapter but then decided to mark it as unread to remove it from your deck/clean it up. It then adds a scrobble event for it.
![image](https://github.com/Kareadita/Kavita/assets/31785430/c6f1c300-22de-4fe8-90fa-06369ba39433)
![image](https://github.com/Kareadita/Kavita/assets/31785430/14b21b09-3611-4e3a-845b-442a9a9a85ff)

**Comment by chilliwow** [comment]
Wasn't this changed/fixed in 0.7.10? I'm still seeing the same thing happening after updating. 
![image](https://github.com/Kareadita/Kavita/assets/31785430/da34e1f7-3b04-4bec-be88-b9051d33cf9e)
![image](https://github.com/Kareadita/Kavita/assets/31785430/a32a5eb2-414f-4ac4-a349-5edaa38093a6)


~~Also with scrobbling if the entry is already completed on AniList, I'm still seeing it in scrobble history.~~ **Edit: Seems like it's just showing in history but doesn't actually change on AniList, it's still under my Completed section and didn't move, which is the desired behavior. It just shows in scrobble history like in the image above**
![image](https://github.com/Kareadita/Kavita/assets/31785430/fa6d4e8c-463f-4f73-ac5d-8960b2a2a1a3)

**Comment by majora2007** [comment]
Correct. It was already in your history so it's not going to remove it. But going forward, it won't create any new events under these conditions.

**Comment by chilliwow** [comment]
Reopening after enabling scrobbling again and not getting the desired behavior.

It should not set an entry to Reading status on AniList if the scrobbling event is Chapter 0.
![image](https://github.com/Kareadita/Kavita/assets/31785430/0be6e99a-4724-42b8-990c-086f004b6688)
![image](https://github.com/Kareadita/Kavita/assets/31785430/a8ef9747-2a1b-45f8-861e-439ad88040fb)
![image](https://github.com/Kareadita/Kavita/assets/31785430/778139a5-dda0-44fd-aff7-c2273f047358)
![image](https://github.com/Kareadita/Kavita/assets/31785430/cf256af7-6862-4a59-9cce-35e0f38f3a96)


Reproduce by just pressing "Mark as Unread" on any series. It will then create a scrobbling event for Chapter 0.
![image](https://github.com/Kareadita/Kavita/assets/31785430/6b867318-8efa-4545-b546-dc6042daf96e)

**Comment by majora2007** [comment]
This was fixed just recently and will be out with v0.8.7.
